src/convertToObject.js

```diff
@@ -6,7 +6,24 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const stylesObject = {};
+
+  const lines = sourceString.split(';');
+
+  for (let i = 0; i < lines.length; i++) {
+    const line = lines[i];
+
+    const indexOfColon = line.indexOf(':');
+
+    if (indexOfColon !== -1) {
+      const property = line.slice(0, indexOfColon);
+      const value = line.slice(indexOfColon + 1);
+
+      stylesObject[property.trim()] = value.trim();
+    }
+  }
```

Comment:
Instead of using a for loop, consider using array iteration methods such as `map` and `filter` to process each line of the style string. This would align with the checklist's recommendation to avoid loops and make use of iteration methods for a more functional approach.